packages/core/src/operators/matchPath/matchPath.operator.ts

```diff
@@ -11,19 +11,22 @@ type MatcherOpts = {
   combiner?: boolean;
 };
 
-export const matchPath = (path: string, opts: MatcherOpts = {}) => (
-  source$: Observable<HttpRequest>
-) =>
+const isRequestMatched = (req: HttpRequest) =>
+  !(req.matchPath && req.matchType);
```

Comment:
I think in one line it would be more readable ✌️

Reply:
I think the opposite :(